axon-server-connector/src/main/java/org/axonframework/axonserver/connector/processor/grpc/PlatformInboundMessage.java

```diff
@@ -18,8 +18,9 @@
 import io.axoniq.axonserver.grpc.control.PlatformInboundInstruction;
 
 /**
- * Created by Sara Pellegrini on 15/03/2018.
- * sara.pellegrini@gmail.com
+ * Supplier of {@link PlatformInboundInstruction}
+ *
+ * @author Sara Pellegrini
  */
 public interface PlatformInboundMessage {
```

Comment:
Question: Isn't this a functional interface?

Reply:
Yes, it is. I added @FunctionalInterface annotation.